tools/clang/test/HLSLFileCheck/passes/dxil/dxil_remove_dead_pass/delete_constant_dce.hlsl

```diff
@@ -0,0 +1,27 @@
+// RUN: %dxc %s -T ps_6_0 -Od | FileCheck %s
+
+// This test verifies the fix for a deficiency in RemoveDeadBlocks where:
```

Comment:
Seems like this is a good candidate to also have a "unit" test for to validate that the DxilRemoveDeadBlocks pass is working as expected.

Reply:
Like an .ll test using opt?

Reply:
Yeah, that's the idea.

Reply:
Done.